packages/ui-menu/src/Menu/index.tsx

```diff
@@ -492,11 +496,11 @@ class Menu extends Component<MenuProps> {
           if (typeof popoverRef === 'function') {
             popoverRef(el)
           }
+          this.ref = el ? el.ref : null
```

Comment:
Popover has an elementRef prop, I used it for Drilldown too. It is easier because the Popover's ref gives back the popover itself: 

```js
        elementRef={(element) => {
          // setting ref for "Popover" version, the popover root
          // (if there is no trigger, we set it in handleDrilldownRef)
          this.handleRef(element)
        }}
        ref={(el) => {
          this._popover = el
          if (typeof popoverRef === 'function') {
            popoverRef(el)
          }
        }}
```

Reply:
OK, I've done it this way